File: AMS_电子对账/code/server/source/com.yss.uco.elecreco.support/src/main/java/com/yss/uco/elecreco/support/bean/ErBbInfo.java
package com.yss.uco.elecreco.support.bean;

import com.yss.framework.api.common.co.AuditableParamPojo;

public class ErBbInfo extends AuditableParamPojo  {
	
	private static final long serialVersionUID = 4366187116066683856L;

	/**
	 * 报文序号
	 */
	private String c_SN = "";

	/**
	 * 报文方向
	 */
	private String c_DV_ER_WAY = "";
	
	/**
	 * 基金代码
	 */
	private String c_ASS_CODE = "";
	
	/**
	 * 组合代码
	 */
	private String c_PORT_CODE = "";
	
	/**
	 * 文件类型
	 */
	private String c_FILE_TYPE = "";
	
	/**
	 * 报表类型
	 */
	private String c_RPT_TYPE = "";
	
	/**
	 * 处理状态
	 */
	private String c_STATE = "";
	
	/**
	 * 对账日期
	 */
	private String d_DATE = "";

	/**
	 * 错误信息
	 */
	private String c_ERR_INFO = "";
	
	/**
	 * 净值确认编号
	 */
	private String c_CONFIRM_ID = "";
	
	/**
	 * 净值确认状态
	 */
	private String c_CONFIRM_EXECUTE = "";
	
	/**
	 * STORY57828需求北京-【新华资产】ams估值核算【高】（关于新华资产系统需支持资管拆分估值表发送实现电子对账的需求）
	 * 托管行代码
	 */
	private String c_TGH_CODE = "";
	
	/**
	 * STORY57828需求北京-【新华资产】ams估值核算【高】（关于新华资产系统需支持资管拆分估值表发送实现电子对账的需求）
	 * 拆分代码
	 */
	private String c_SPLIT_CODE = "";
	
	/**
	 * 手工原因
	 */
	private String c_DV_RESULT = "";
	
	/**
	 * 说明
	 */
	private String c_SUMMARY = "";
	
	/**
	 * STORY58759嘉实基金-电子对账-电子对账管理界面净值确认按钮修改
	 * 净值锁定状态
	 */
	private String c_LOCK_EXECUTE = "";
	
	/**
	 * STORY #104480 净值确认管理优化整合需求对电子对账管理的影响评估优化
	 */
	private int n_JZAUDIT_STATE = 0; 
	
	/**
	 * STORY #109004 【公共需求】电子对账管理中增加显示“反馈联系人”列
	 */
	private String c_DEALER;
	
	public int getN_JZAUDIT_STATE() {
		return n_JZAUDIT_STATE;
	}

	public void setN_JZAUDIT_STATE(int n_JZAUDIT_STATE) {
		this.n_JZAUDIT_STATE = n_JZAUDIT_STATE;
	}

	public String getC_LOCK_EXECUTE() {
		return c_LOCK_EXECUTE;
	}

	public void setC_LOCK_EXECUTE(String c_LOCK_EXECUTE) {
		this.c_LOCK_EXECUTE = c_LOCK_EXECUTE;
	}
	
	
	public String getC_TGH_CODE() {
		return c_TGH_CODE;
	}

	public void setC_TGH_CODE(String c_TGH_CODE) {
		this.c_TGH_CODE = c_TGH_CODE;
	}

	public String getC_SPLIT_CODE() {
		return c_SPLIT_CODE;
	}

	public void setC_SPLIT_CODE(String c_SPLIT_CODE) {
		this.c_SPLIT_CODE = c_SPLIT_CODE;
	}
	public String getC_DV_RESULT() {
		return c_DV_RESULT;
	}

	public void setC_DV_RESULT(String cDVRESULT) {
		this.c_DV_RESULT = cDVRESULT;
	}

	public String getC_SUMMARY() {
		return c_SUMMARY;
	}

	public void setC_SUMMARY(String cSUMMARY) {
		this.c_SUMMARY = cSUMMARY;
	}

	public String getC_PORT_CODE() {
		return c_PORT_CODE;
	}

	public void setC_PORT_CODE(String c_PORT_CODE) {
		this.c_PORT_CODE = c_PORT_CODE;
	}
	
	public String getC_ERR_INFO() {
		return c_ERR_INFO;
	}

	public void setC_ERR_INFO(String cERRINFO) {
		c_ERR_INFO = cERRINFO;
	}


	public String getD_DATE() {
		return d_DATE;
	}


	public void setD_DATE(String dDATE) {
		d_DATE = dDATE;
	}


	public String getC_SN() {
		return c_SN;
	}


	public void setC_SN(String cSN) {
		c_SN = cSN;
	}


	public String getC_DV_ER_WAY() {
		return c_DV_ER_WAY;
	}


	public void setC_DV_ER_WAY(String cDVERWAY) {
		c_DV_ER_WAY = cDVERWAY;
	}

	public String getC_ASS_CODE() {
		return c_ASS_CODE;
	}

	public void setC_ASS_CODE(String cASSCODE) {
		c_ASS_CODE = cASSCODE;
	}


	public String getC_FILE_TYPE() {
		return c_FILE_TYPE;
	}


	public void setC_FILE_TYPE(String cFILETYPE) {
		c_FILE_TYPE = cFILETYPE;
	}


	public String getC_RPT_TYPE() {
		return c_RPT_TYPE;
	}


	public void setC_RPT_TYPE(String cRPTTYPE) {
		c_RPT_TYPE = cRPTTYPE;
	}


	public String getC_STATE() {
		return c_STATE;
	}


	public void setC_STATE(String cSTATE) {
		c_STATE = cSTATE;
	}

	
	public String getC_CONFIRM_ID() {
		return c_CONFIRM_ID;
	}

	public void setC_CONFIRM_ID(String c_CONFIRM_ID) {
		this.c_CONFIRM_ID = c_CONFIRM_ID;
	}

	public String getC_CONFIRM_EXECUTE() {
		return c_CONFIRM_EXECUTE;
	}

	public void setC_CONFIRM_EXECUTE(String c_CONFIRM_EXECUTE) {
		this.c_CONFIRM_EXECUTE = c_CONFIRM_EXECUTE;
	}

	public String getC_DEALER() {
		return c_DEALER;
	}

	public void setC_DEALER(String c_DEALER) {
		this.c_DEALER = c_DEALER;
	}
	 
}

File: AMS_电子对账/code/client/source/YssElecReco/Pojo/Er/ErBbInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Newtonsoft.Json;
using FAST.Common.Service.Pojo.Base;

namespace YssElecReco.Pojo.Er
{
    /// <summary>
    /// 电子对账结果实体对象
    /// </summary>
    public class ErBbInfo : AuditableParamPojo
    {
        /// <summary>
        /// 报文序号
        /// </summary>
        private string c_SN = "";

        /// <summary>
        /// 手工原因
        /// </summary>
        private string c_DV_RESULT = "";

        /// <summary>
        /// 说明
        /// </summary>
        private string c_SUMMARY = "";

        /// <summary>
        /// 日期
        /// </summary>
        private string c_DV_ER_WAY = "";

        /// <summary>
        /// 发送状态
        /// </summary>
        private string c_STATE = "";

        /// <summary>
        /// 文件类型
        /// </summary>
        private string c_FILE_TYPE = "";

        /// <summary>
        /// 组合代码
        /// </summary>
        private string c_PORT_CODE = "";

        /// <summary>
        /// 报表类型
        /// </summary>
        private string c_RPT_TYPE = "";

        /// <summary>
        /// 基金代码
        /// </summary>
        private string c_ASS_CODE = "";

        /// <summary>
        /// 对账日期
        /// </summary>
        private string d_DATE = "";

        /// <summary>
        /// 错误信息
        /// </summary>
        private string c_ERR_INFO = "";

        /// <summary>
        /// 托管机构
        /// </summary>
        private string c_TGH_CODE = "";

        /// <summary>
        /// 拆分代码
        /// </summary>
        private string c_SPLIT_CODE = "";

        /// <summary>
        /// 净值审核
        /// </summary>
        private int n_JZAUDIT_STATE = 0;

        /// <summary>
        /// 托管行处理人
        /// </summary>
        private string c_DEALER = "";

        /// <summary>
        /// 托管行处理人
        /// </summary>
        [JsonProperty(PropertyName = "c_DEALER")]
        public string C_DEALER
        {
            set { c_DEALER = value; }
            get { return c_DEALER; }
        }

        /// <summary>
        /// 净值审核状态
        /// </summary>
        [JsonProperty(PropertyName = "n_JZAUDIT_STATE")]
        public int N_JZAUDIT_STATE
        {
            get { return n_JZAUDIT_STATE; }
            set { n_JZAUDIT_STATE = value; }
        }

        /// <summary>
        /// 属性: 托管机构
        /// </summary>
        [JsonProperty(PropertyName = "c_TGH_CODE")]
        public string C_TGH_CODE
        {
            set { c_TGH_CODE = value; }
            get { return c_TGH_CODE; }
        }

        /// <summary>
        /// 属性: 拆分代码
        /// </summary>
        [JsonProperty(PropertyName = "c_SPLIT_CODE")]
        public string C_SPLIT_CODE
        {
            set { c_SPLIT_CODE = value; }
            get { return c_SPLIT_CODE; }
        }
		
        /// <summary>
        /// 属性: C_DV_RESULT 
        /// </summary>
        [JsonProperty(PropertyName = "c_DV_RESULT")]
        public string C_DV_RESULT
        {
            set { c_DV_RESULT = value; }
            get { return c_DV_RESULT; }
        }

        /// <summary>
        /// 属性: C_SUMMARY 
        /// </summary>
        [JsonProperty(PropertyName = "c_SUMMARY")]
        public string C_SUMMARY
        {
            set { c_SUMMARY = value; }
            get { return c_SUMMARY; }
        }


        /// <summary>
        /// 属性: 组合代码
        /// </summary>
        [JsonProperty(PropertyName = "c_PORT_CODE")]
        public string C_PORT_CODE
        {
            set { c_PORT_CODE = value; }
            get { return c_PORT_CODE; }
        }
        

        /// <summary>
        /// 属性: C_ERRINFO 
        /// </summary>
        [JsonProperty(PropertyName = "c_ERR_INFO")]
        public string C_ERR_INFO
        {
            set { c_ERR_INFO = value; }
            get { return c_ERR_INFO; }
        }

        /// <summary>
        /// 属性: d_DATE 
        /// </summary>
        [JsonProperty(PropertyName = "d_DATE")]
        public string D_DATE
        {
            set { d_DATE = value; }
            get { return d_DATE; }
        }

        /// <summary>
        /// 属性: c_SN 
        /// </summary>
        [JsonProperty(PropertyName = "c_SN")]
        public string C_SN
        {
            set { c_SN = value; }
            get { return c_SN; }
        }

        /// <summary>
        /// 属性: c_DV_ER_WAY
        /// </summary>
        [JsonProperty(PropertyName = "c_DV_ER_WAY")]
        public string C_DV_ER_WAY
        {
            set { c_DV_ER_WAY = value; }
            get { return c_DV_ER_WAY; }
        }

        /// <summary>
        /// 属性: c_STATE
        /// </summary>
        [JsonProperty(PropertyName = "c_STATE")]
        public string C_STATE
        {
            set { c_STATE = value; }
            get { return c_STATE; }
        }

        /// <summary>
        /// 属性: c_FILE_TYPE
        /// </summary>
        [JsonProperty(PropertyName = "c_FILE_TYPE")]
        public string C_FILE_TYPE
        {
            set { c_FILE_TYPE = value; }
            get { return c_FILE_TYPE; }
        }

        /// <summary>
        /// 属性: c_RPT_TYPE
        /// </summary>
        [JsonProperty(PropertyName = "c_RPT_TYPE")]
        public string C_RPT_TYPE
        {
            set { c_RPT_TYPE = value; }
            get { return c_RPT_TYPE; }
        }

        /// <summary>
        /// 属性: c_PORT_CODE
        /// </summary>
        [JsonProperty(PropertyName = "c_ASS_CODE")]
        public string C_ASS_CODE
        {
            set { c_ASS_CODE = value; }
            get { return c_ASS_CODE; }
        }

        /// <summary>
        /// 属性：净值确认编号
        /// </summary>
        [JsonProperty(PropertyName = "c_CONFIRM_ID")]
        public string C_CONFIRM_ID { get; set; }

        /// <summary>
        /// 属性：净值确认执行状态
        /// </summary>
        [JsonProperty(PropertyName = "c_CONFIRM_EXECUTE")]
        public string C_CONFIRM_EXECUTE { get; set; }

        /// <summary>
        /// STORY58759嘉实基金-电子对账-电子对账管理界面净值确认按钮修改
        /// 属性：净值锁定执行状态
        /// </summary>
        [JsonProperty(PropertyName = "c_LOCK_EXECUTE")]
        public String C_LOCK_EXECUTE { get; set; }
    }
}
